docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -600,19 +603,20 @@ Emitted when operation `id` is cancelled.
 
 Emitted when the minimum delay for future operations is modified.
 
-== Utils
+== Votes
+
+The `VotesComponent` provides a flexible system for tracking voting power and delegation. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance.
```

Comment:
```suggestion
The `VotesComponent` provides a flexible system for tracking voting power. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance.
```

Reply:
why not mention delegation? Seems relevant

Reply:
It is mentioned in the next sentence, while in this one it sounds like it is also a "flexible system for tracking delegation" (the and), while the main purpose is to track voting power, while delegation is a feature.